src/cloud-cli/applications/deploy-app-code.ts

```diff
@@ -69,7 +47,38 @@ export async function deployAppCode(host: string): Promise<number> {
       }
     );
     const deployOutput = response.data as DeployOutput;
-    logger.info(`Successfully deployed ${appName} with version ${deployOutput.ApplicationVersion}`);
+    logger.info(`Submitted deploy request for ${appName}. Assigned version: ${deployOutput.ApplicationVersion}`);
+
+    // Wait for the application to become available
+    let count = 0
+    let applicationAvailable = false
+    while (!applicationAvailable) {
+      count += 1
+      if (count % 5 === 0) {
+        logger.info(`Waiting for ${appName} with version ${deployOutput.ApplicationVersion} to be available`);
+        if (count > 20) {
+          logger.info(`If ${appName} takes too long to become available, check its logs at...`);
```

Comment:
I assume this is a placeholder?

Reply:
Yeah, until we have a functioning logs command